lib/DataObject/FielddefinitionMarshaller/Block/Consent.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @category   Pimcore
+ * @package    Object
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\DataObject\FielddefinitionMarshaller\Block;
```

Comment:
What do you think about `Pimcore\DataObject\BlockDataMarshaller`? 
I'm just thinking because this marshallers do not actually marshal the field-definitions, but the data of fields within a block data-type. 🤔